src/Windows/dcap_provider.sln

```diff
@@ -37,6 +39,12 @@ Global
 		{C356FCD3-13E4-40F1-9FDD-D2E68FA97885}.Release|x64.ActiveCfg = Release|x64
 		{C356FCD3-13E4-40F1-9FDD-D2E68FA97885}.Release|x64.Build.0 = Release|x64
 		{C356FCD3-13E4-40F1-9FDD-D2E68FA97885}.Release|x86.ActiveCfg = Release|x64
+		{A5E8BFB6-3DBE-498F-9431-544D7C09B1F0}.Debug|x64.ActiveCfg = Debug|x64
+		{A5E8BFB6-3DBE-498F-9431-544D7C09B1F0}.Debug|x64.Build.0 = Debug|x64
+		{A5E8BFB6-3DBE-498F-9431-544D7C09B1F0}.Debug|x86.ActiveCfg = Debug|x64
+		{A5E8BFB6-3DBE-498F-9431-544D7C09B1F0}.Release|x64.ActiveCfg = Release|x64
+		{A5E8BFB6-3DBE-498F-9431-544D7C09B1F0}.Release|x64.Build.0 = Release|x64
+		{A5E8BFB6-3DBE-498F-9431-544D7C09B1F0}.Release|x86.ActiveCfg = Release|x64

```

Comment:
Why are we building a 32bit version of the provider?

Reply:
Thank you removed. We wouldn't have been able to build this since the .vcsproj doesn't have the configuration defined